service/src/test/kotlin/no/nav/su/se/bakover/service/søknad/lukk/LukkSøknadServiceImplTest.kt

```diff
@@ -66,17 +66,7 @@ internal class LukkSøknadServiceImplTest {
             fnr = fnr,
             aktørId = AktørId(aktørId = "123")
         ),
-        navn = Navn(fornavn = "Tore", mellomnavn = "Johnas", etternavn = "Strømøy"),
-        telefonnummer = null,
-        adresse = null,
-        statsborgerskap = null,
-        kjønn = null,
-        fødselsdato = null,
-        adressebeskyttelse = null,
-        skjermet = null,
-        kontaktinfo = null,
-        vergemål = null,
-        fullmakt = null,
+        navn = Navn(fornavn = "Tore", mellomnavn = "Johnas", etternavn = "Strømøy")
```

Comment:
Kunne i tillegg kanskje dratt person ut i en testUtil, siden vi sjelden endrer på verdiene :shrug: Kan være en funksjon som tar inn fnr f.eks. med en default.

Noen plasser har vi person i testutil allerede

Reply:
Hvor har vi person i testutils? My search foo is weak.

Reply:
Ja, det finnes vel ikke for alle moduler, men vi har en BehandlingTestUtils i service som kunne vært flytta og gjort litt mer generisk